plugins/module_utils/ansible_freeipa_module.py

```diff
@@ -239,6 +239,8 @@ def compare_args_ipa(module, args, ipa):  # noqa
     base_debug_msg = "Ansible arguments and IPA commands differed. "
 
     for key in args.keys():
+        if key in ignore:
+            continue
```

Comment:
I suggest removing the `continue` test and statement and create a search list with `[k for k in args.keys() if k not in ignore]` and using it in the `for`.

Reply:
Good idea, I have done what you suggested, thanks for the feedback.

Reply:
Morning, @rjeffman You happy with the changes that were made?